CHANGELOG.md

```diff
@@ -4,16 +4,15 @@ All notable changes to the "azure-cosmosdb" extension will be documented in this
 
 ## 0.15.0 - 2020-09-14
 ### Added
-- PostgreSQL support to Attached Accounts
-- PostgreSQL support to API
+- Attach PostgreSQL servers by connection string
+- PostgreSQL support for integration with App Service extension
+### Fixed
```

Comment:
Take a close look at whitespace and keep that consistent as well. Sometimes that can make a difference in markdown. For example, we usually do an empty line between each "### Added" and "### Fixed" section. Also, I don't think a link works if there's a space between `[Bugs fixed]` and `(https://...`